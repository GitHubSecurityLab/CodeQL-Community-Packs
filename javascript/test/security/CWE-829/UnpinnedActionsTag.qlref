security/CWE-829/UnpinnedActionsTag.ql
