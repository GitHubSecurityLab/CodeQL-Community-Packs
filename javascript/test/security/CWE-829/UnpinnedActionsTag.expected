| .github/workflows/unpinned-action.yml:6:7:9:4 | name: C ... ository | Unpinned 3rd party Action 'Unpinned Action Example' step $@ uses 'actions-third-party-mirror/checkout' with ref 'v3', not a pinned commit hash | .github/workflows/unpinned-action.yml:6:7:9:4 | name: C ... ository | name: C ... ository |
| .github/workflows/unpinned-unnamed-action.yml:4:7:7:4 | name: C ... ository | Unpinned 3rd party Action 'unpinned-unnamed-action.yml' step $@ uses 'actions-third-party-mirror/checkout' with ref 'v3', not a pinned commit hash | .github/workflows/unpinned-unnamed-action.yml:4:7:7:4 | name: C ... ository | name: C ... ository |
